README.md

```diff
@@ -2,6 +2,17 @@ Mautic Introduction
 ===========
 ![Mautic](https://www.mautic.org/media/images/github_readme.png "Mautic Open Source Marketing Automation")
 
+## Supported Versions
+Please note that the release dates indicated with * below are *estimations*, no rights can be derived from them.
+
+| Branch | Beta Release | Initial Release | Active Support Until | Security Support Until
+|--|--|--|--|--|
+|2.15  | 27 Sep 2019 | 8 Oct 2019 | 8 Oct 2019 | 8 Oct 2019
```

Comment:
This says that 2.15 is actively supported only until 8th October 2019, is that correct @dennisameling @alanhartless or is this an error in the dates?

Reply:
@RCheesley not sure. I was assuming that there won't be another 2.15 release, therefore mentioned that active support is only until October 8th 2019.